llama-index-core/llama_index/core/tools/function_tool.py

```diff
@@ -21,6 +21,16 @@ async def _async_wrapped_fn(*args: Any, **kwargs: Any) -> Any:
     return _async_wrapped_fn
 
 
+def async_to_sync(func_async: AsyncCallable) -> Callable:
+    """Async from sync."""
+
+    def _sync_wrapped_fn(*args: Any, **kwargs: Any) -> Any:
+        loop = asyncio.get_event_loop()
+        return loop.run_until_complete(func_async(*args, **kwargs))
```

Comment:
Should we use `asyncio.run_coroutine_threadsafe` instead?

Reply:
Lets update `asyncio_run()` in `llama_index.core.async_utils` if needed in another PR